<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>434</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>434</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="edit_username">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>365</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
font: 75 15pt &quot;等线&quot;;
background:transparent;
border-width:0;
border-style:outset
</string>
   </property>
   <property name="placeholderText">
    <string>请输入6-20位的英文或数字</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_password">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>445</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
font: 75 15pt &quot;等线&quot;;
background:transparent;
border-width:0;
border-style:outset
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入6-20位的英文或数字</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_register">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>540</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background:transparent;
border-width:0;
border-style:outset
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>434</width>
     <height>768</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/base/注册界面图.png</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>edit_username</zorder>
  <zorder>edit_password</zorder>
  <zorder>btn_register</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
